features/ultils/world.ts

```diff
@@ -36,6 +60,40 @@ export class OurWorld extends World {
       destFilePath: filePath,
     });
   }
+
+  public getCredentialByAppKey(appKey: string): Credential {
+    const credential = getCredentialByAppKey(this.credentials, appKey);
+    if (credential === undefined) {
+      throw new Error(`The credential with app key ${appKey} is not found`);
+    }
+
+    if (!credential.appId) {
+      throw new Error(`The credential with app key ${appKey} has no App ID`);
+    }
```

Comment:
MUST:
Is it for checking if `appId` is an empty string?
If so, please use `.length`.

```suggestion
    if (credential.appId.length === 0) {
      throw new Error(`The credential with app key ${appKey} has no App ID`);
    }
```